flink_jobs_v2/batch_status/src/main/java/argo/batch/CalcEndpointTimeline.java

```diff
@@ -104,7 +104,7 @@ public void reduce(Iterable<StatusTimeline> in, Collector<StatusTimeline> out) t
             }
         }
 
-        TimelineAggregator timelineAggregator = new TimelineAggregator(timelinelist, this.opsMgr.getDefaultExcludedInt());
+        TimelineAggregator timelineAggregator = new TimelineAggregator(timelinelist);
```

Comment:
Do we need this changed? See similar changes in the rest of the diff

Reply:
Some issues with git , not appearing it as confict. Will check better in the next PRs